Meetings/devel-call-2024-09-04.md

```diff
@@ -0,0 +1,27 @@
+# Meeting Minutes: SVSM Development Call (September 4th, 2024)
+
+## Attendees:
+
+Adam Dunlap, Diego Gonzalez Villalobos, Dionna Glaze, Hajira Zaman, Huibo Wang, James Bottomley, Joerg Roedel, Nicolai Stange, Oliver Steffen, Peter Fang, Rama AMD, Roy Hopkins, Stefano Garzarella, Tom Dohrmann, Vijay Dhanraj
+
+## Topics:
+
+### Early Attestation Process Document
+
```

Comment:
What about add a link to the document?

https://docs.google.com/document/d/11ZsxP8jsviP3ddp9Hrn0rf6inttNw_Pbnz0psXlxlPs

Reply:
Link is added.